watchtower-plugin/tests/conftest.py

```diff
@@ -2,7 +2,7 @@
 import subprocess
 
 from pyln.testing.fixtures import *  # noqa: F401,F403
-from pyln.testing.utils import DEVELOPER, BITCOIND_CONFIG, TailableProc
+from pyln.testing.utils import BITCOIND_CONFIG, TailableProc
```

Comment:
There are still some instances of `DEVELOPER` in this file, alongside a marker to check that developer is enabled